### PR TITLE
Add scrobble_filter to provide a configurable list of URI schemes not…

### LINKED ISSUE #28: Need URI filters for scrobbles
Scrobbling from Spotify to Last.FM is now done by a web 'app' - you simply enable scrobbling on your Spotify account and all tracks you play on Spotify on any device are automatically scrobbled. 

This creates an issue if you're using Mopidy-Scrobbler in that all Spotify tracks  played through Mopidy now get scrobbled twice - once by Mopidy-Scrobbler and once by Spotify. 

We need a list of uri schemes to NOT scrobble so that people can enable Spotify web scrobbling but still scrobble local tracks using Mopidy-Scrobbler.

### PR DESCRIPTION
**Pull request by fatg3erman**
… not scrobble

This should fix #28, however I've been testing it today and found that even with the Spotify web app scrobbling enabled, I don't see double scrobbles from Mopidy even if mopidy is also scrobbling.

Not sure why this is, perhaps it's only temporary, but adding this as a config value dosn't do any harm.

### CODE REVIEW

**Comment by fatg3erman** [comment]
The failure appears to be something in pylast that I haven't changed, so I don't know what's failing. It runs on my system.

File: README.rst [hunk 1]
```diff
@@ -41,12 +41,14 @@ found at ``~/.config/mopidy/mopidy.conf``::
     [scrobbler]
     username = alice
     password = secret
+    scrobble_filer = list
```
Comment (jjok) [inline README.rst]: `scrobble_filter`?

**Comment by jjok** [comment]
Looks good. I think I would call it "blacklist" though, to make it clearer that those extensions won't be scrobbled. It also means that you can add a "whitelist" at some point too.

Maybe `backend_blacklist` or `extension_blacklist`?

**Comment by fatg3erman** [comment]
I agree. backend_blacklist is better. It's still failing the checks though but I don't know why. It's running fine for me.

File: README.rst [hunk 2]
```diff
@@ -41,12 +41,14 @@ found at ``~/.config/mopidy/mopidy.conf``::
     [scrobbler]
     username = alice
     password = secret
+    backend_blacklist = list
```
Comment (djmattyg007) [inline README.rst:44]: I think it's worth considering a word other than "blacklist". How about "ignorelist"?

File: mopidy_scrobbler/ext.conf [hunk 3]
```diff
@@ -2,3 +2,4 @@
 enabled = true
 username =
 password =
+backend_blacklist =
```
Comment (djmattyg007) [inline mopidy_scrobbler/ext.conf:5]: Looks like you're missing a newline at the end of the file here.

File: README.rst (hunk 2 above)
Comment (jodal) [reply README.rst:44]: Agree.

We also have some "ignore"/"exclude" lists in the config of other extensions, like Mopidy-Local and Mopidy-Files. I'd like this to be aligned with the naming there.

**Pull request closed without merging**